### ISSUE TITLE
Unable to exclude folders from tslint

### ISSUE THREAD

**Issue by ltfschoen** [polkadot-js/apps#149]
If I try the following it still shows linting errors from the respective folders
```
./node_modules/.bin/tslint --project . --exclude "node_modules/build/**/*.ts" --exclude "node_modules/csstype/**/*.ts" --exclude "node_modules/semantic-ui-react/**/*.d.ts" && tsc --noEmit
```

Likewise, if I update tsconfig.json with the following it still shows linting errors from the respective folders to be excluded:
```
{
  "extends": "./node_modules/@polkadot/dev/config/tsconfig",
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      ...
    },
    "typeRoots": [
      ...
    ]
  },
  "linterOptions": {
    "exclude": [
      "node_modules/build/**/*.{j,t}s",
      "node_modules/csstype/**/*.{j,t}s",
      "node_modules/semantic-ui-react/**/*.{j,t}s"
    ]
  }
}
```

**Comment by jacogr** [comment]
Merge in master where the typescript version is fixed to 2.9.2. Apart from not appearing here, it also doesn't appear on CI so it is a local issue.

Only place where this is an issue is `build/` - that would be useful to exclude from the actual tsconfig (not linting) step. That can be fixed by 

```
"exclude": [
    "node_modules",
    "build"
  ]
```

**Comment by ltfschoen** [comment]
Your absolutely right. I didn't pull latest changes yesterday due to NPM issues, but now that I have today the linting works fine without any changes required
